test/lib/userProfiles.js

```diff
@@ -29,7 +29,7 @@ const addBookmarks4000 = addBookmarksN(4000)
 const addTabsN = function (total) {
   return function * (client) {
     const data = []
-    const buffer = new Buffer(2)
+    const buffer = Buffer.from(2)
```

Comment:
I think this should be alloc(2) here actually, ditto above

Reply:
so `Buffer.alloc(2)`

Reply:
my bad, fixed in bb324eb12e94eb9f75b82d3eda5364abeb2a233b